spec/javascripts/app/helpers/text_formatter_spec.js

```diff
@@ -261,7 +270,7 @@ describe("app.helpers.textFormatter", function(){
       it("doesn't crash Firefox", function() {
         var content = "antifaschistisch-feministische ://";
         var parsed = this.formatter(content);
-        expect(parsed).toContain(content);
+        expect(parsed).toContain(content);nintendo 
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.